backend/app/rest/api/rest_public.go

```diff
@@ -447,6 +451,80 @@ func (s *public) loadPictureCtrl(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// GET /search?site=siteID&query=queryText&limit=20&skip=10 - search documents
+func (s *public) searchQueryCtrl(w http.ResponseWriter, r *http.Request) {
```

Comment:
This tested only partly, I think it's worth effort improving the coverage.

Reply:
I've improve coverage